### PR TITLE
Provide example to utilize k8s_external plugin

### PR DESCRIPTION
**Pull request by Timoses**
The example provides a specific use case of k8s_external and may help others to grasp `k8s_external`'s capabilities.

### 1. Why is this pull request needed and what does it do?
For me as a DNS newbie it is quite challenging to read the description of the plugin. I was unsure what it actually provided.

### 2. Which issues (if any) are related?
n/a

### 3. Which documentation changes (if any) need to be made?
n/a

### 4. Does this introduce a backward incompatible change or deprecation?
No

### CODE REVIEW

File: plugin/k8s_external/README.md [hunk 1]
```diff
@@ -68,6 +68,23 @@ Enable names under `example.org` to be resolved to in cluster DNS addresses.
 }
 ~~~
 
+Create a service in Kubernetes.
```
Comment (chrisohaver) [inline plugin/k8s_external/README.md]:
```suggestion
With the Corefile above, the following Service will get an `A` record for `test.default.example.org` with IP address `192.168.200.123`.
```

File: plugin/k8s_external/README.md [hunk 2]
```diff
@@ -68,6 +68,23 @@ Enable names under `example.org` to be resolved to in cluster DNS addresses.
...
+ name: test
+ namespace: default
+spec:
+ clusterIP: None
+ externalIPs:
+ - 192.168.200.123
+ type: ClusterIP
+~~~
+
+This will create a resource record for `test.default.example.org`.
```
Comment (chrisohaver) [inline plugin/k8s_external/README.md]:
```suggestion
```

**Review by chrisohaver — CHANGES_REQUESTED** [review]
Thanks, suggesting some rewording/reorganizing...

**Comment by Timoses** [comment]
Do you want me to squash them manually?

**Comment by miekg** [comment]
We usually squash on merge, but you can squash yourself not you prefer

**Review by chrisohaver — APPROVED** [review]

**Pull request merged**